GOVERNANCE.md

```diff
@@ -1,3 +1,187 @@
+in additional committers to help with the review or provide additional
+thoughts.  A committer who authors or merges a PR is typically
+responsible for any testing fallout caused by that PR, typically by
+involving the PR's author when it was not theirs.  In some cases, the
+reviewing committer or another volunteer may step up to help out.
+
+At present, the list of committers includes all of the
+technical leaders listed previously, as well as:
+
+* [Jade Abraham](https://github.com/jabraham17)
+* [Soohoon Choi](https://github.com/soohoonchoi)
+* [Matt Drozt](https://github.com/MattToast)
+* [Daniel Fedorin](https://github.com/DanilaFe)
+* [Michael Ferguson](https://github.com/mppf)
+* [Abhishek Girish](https://github.com/agirish)
+* [Shreyas Khandekar](https://github.com/ShreyasKhandekar)
+* [Vass Litvinov](https://github.com/vasslitvinov)
+* [David Longnecker](https://github.com/dlongnecke-cray)
+* [Ben McDonald](https://github.com/bmcdonald3)
+* [Ahmad Rezaii](https://github.com/arezaii)
+* [Anna Rift](https://github.com/riftEmber)
+* [Elliot Ronaghan](https://github.com/ronawho)
+* [Michelle Strout](https://github.com/mstrout)
+* [Karlon West](https://github.com/karlonw)
+
+
+Decision-Making
+---------------
+
+Generally speaking, decision-making on the Chapel project is done by
+the technical leadership and committers in this section on a consensus
+basis, taking the opinions of end-users into account when the decision
+is likely to affect them.  Project-wide decisions tend to involve all
+technical leads and committers, while those that are specific to a
+given effort can be made by a subteam lead and its members, keeping
+the tech lead apprised of key decisions.
+
+Votes are rarely taken in a binding manner, but are often used to
+gauge the level of support for an idea or direction.  In such votes,
+those in the minority are typically given the opportunity to continue
+to push for alternative approaches, or to concede to the majority,
+having registered their opinion.
+
+For non-trivial decisions in language design, library APIs, or code
+architecture, the project or subteam typically forms an ad hoc subteam
+including at least three committers to explore the topic and propose a
+solution, where anyone on the broader team is encouraged to
+participate if they feel interested or invested.  Once the ad hoc
+subteam has reached a conclusion, it is summarized back to the full
+team.  Generally speaking, those who did not join the ad hoc subteam
+are expected to go along with its decision, unless they have an
+insight or strong reason that the proposal is untenable—particularly
+if it involves factors the subteam was not, or could not have been,
+aware of.  In practice, this has happened only very rarely.
+
+Decisions that relate to the design of the language or libraries are
+captured and discussed on public [GitHub
+issues](https://github.com/chapel-lang/chapel/issues?q=is%3Aissue) as
```

Comment:
maybe link or verbally refer to the "Design" label?

Reply:
I considered that, but ended up not doing it, thinking it might cast too narrow of a net (i.e., that there may be such decisions that don't get the label but still fall into what's being described here)?  

Then again, it's not as though this needs to be the definitive list, more of an illustration of the concept, so maybe better to focus on a subset that clearly illustrate it?

Reply:
My mild preference is because creating a GitHub repo and throwing in bunch of issues is easy. It would be good to demonstrate that it is not just a heap of issues, but there's organization in how we handle those, which is a good thing for an open source project. Saying these, I kind of convinced myself that a verbal reference is also great. I don't know how many people will click on the link and observe labels etc. IOW, you can (and maybe should) keep the link as-is but add a verbal reference to the label?